odooTimereg.py

```diff
@@ -40,7 +40,9 @@ def hours(self, client, projectids, from_date=None, to_date=None):
             ('project_id', 'in', projectids),
             ('date', '>=', from_date.strftime('%Y-%m-%d')),
             ('date', '<', to_date.strftime('%Y-%m-%d')),
+            '|',
             ('timesheet_invoice_type', '!=', 'non_billable_project'),
+            ('timesheet_invoice_type', '=', False),
```

Comment:
Please leave a comment here describing which part is required for pre-2020, and which is valid for the future.

Reply:
Done